server/services/damsService/damsService.js

```diff
@@ -137,9 +161,70 @@ async function getEnsembleImageUrl(ensembleIndex) {
   return ensembleImageUrl;
 }
 
-module.exports = {
-  getAssetByObjectNumber,
+/** Retrieves all archive assets from NetX. Archives are objects in NetX that have "Object Type (TMS)" set to "Archive Asset"
+ * As of the present, there are about 82 archive assets in NetX, though they may each be associated with 1-or-more artwork assets.
+ * This relation can be seen in the "folders" field of the archive assets response.
+ *
+ * This function will query NetX to retrieve all archive assets - not filtering for any particular Object Number/Object Id
+ * It will return the archive assets in a map with the identified Object Number and the archives associated to that
+ *
+ * @returns {Promise<{}>} - A map of the Object IDs to their asset list
+ */
+async function getAllArchiveAssets() {
```

Comment:
This is the net new addition to this file - just retrieving the archive assets from etX and returning them in a map of Object Number to archive assets